packages/back-end/src/models/ExperimentSnapshotModel.ts

```diff
@@ -219,6 +221,25 @@ export async function updateSnapshot({
   });
   if (!experimentSnapshotModel) throw "Internal error";
 
+  const shouldUpdateExperimentAnalysisSummary =
+    experimentSnapshotModel.type === "standard" &&
+    experimentSnapshotModel.status !== "running";
```

Comment:
Should we have additional checks for latest phase (I think you can refresh a previous phase's snapshot)?

Reply:
Oh, great call. Yeah I think you can.